docs/sources/mimir/references/architecture/components/alertmanager.md

```diff
@@ -156,3 +156,66 @@ You must configure the [ruler]({{< relref "./ruler" >}}) with the addresses of A
 Point the address to Alertmanager’s API.
 You can configure Alertmanager’s API prefix via the `-http.alertmanager-http-prefix` flag, which defaults to `/alertmanager`.
 For example, if Alertmanager is listening at `http://mimir-alertmanager.namespace.svc.cluster.local` and it is using the default API prefix, set `-ruler.alertmanager-url` to `http://mimir-alertmanager.namespace.svc.cluster.local/alertmanager`.
+
+## Enable UTF-8
+
+In a wide effort to support alerts from [OpenTelemetry](https://opentelemetry.io/) (OTel) data, [Prometheus Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) has added support for UTF-8. This is supported as an opt-in feature for the Grafana Mimir Alertmanager in Mimir versions 2.12 and later, and will be changed to opt-out in future versions of Mimir.
```

Comment:
Oh OK. I think we want some language here that encourages users to opt-in before it is changed into an opt-out, because as is written further down, some users might have issues with some of the breaking changes, and we don't want it to be a surprise for them. How best should we word it?

Reply:
I'll remove it, and we can add it back later if we need to 👍

Reply:
I think it's still best to just leave it as opt-in for now while still encouraging people to do it.

Reply:
We can encourage people to opt-in without mentioning that the setting will change in future releases.